docs/articles/expensify-classic/expensify-card/Request-the-Card.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Request the Card
+description: Details on requesting the Expensify Card as an employee
+---
+# Overview
+
+Once your organization is approved for the Expensify Card, you can request a card! 
+
+This article covers how to request, activate, and replace your physical and virtual Expensify Cards.
+
+# How to get your first Expensify Card
+
+An admin in your organization must first enable the Expensify Cards before you can receive a card. After that, an admin may assign you a card by setting a limit. You can think of setting a card limit as “unlocking” access to the card.
+
+If you haven’t been assigned a limit yet, look for the task on your account's homepage that says, “Ask your admin for the card!”. This task allows you to message your admin team to make that request.
```

Comment:
```suggestion
If you haven’t been assigned a limit yet, look for the task on your account's homepage that says, “Ask your admin for the card!” This task allows you to message your admin team to make that request.
```

Reply:
Oooooh im so glad you're asking about this. Are you saying that we should not include `"` in copy on helpdot? if so, i think we could standardize on italicizing instead of using quotation marks. lmk your thoughts!

Reply:
Sorry I'm saying to use `"` instead of `”` (notice the slant lol, I know it's subtle). We could also just handle this as cleanup at the very end with a bulk search/replace operation in the directory 🤷‍♀️

Reply:
> We could also just handle this as cleanup at the very end with a bulk search/replace operation in the directory
i think this makes sense. 

also, how does one even make a slanted `"`? lol.

Reply:
Haha, some editors convert them automatically (e.g. Google Docs) so they sneak in here from copy/paste.

And okay definitely agree a mass search/replace at the end is going to be easiest. Feel free to tag me in when the time comes if you want me to handle it! (but @rushatgabhane is definitely capable too 😄)